Networking/Networking/Model/Copiable/Models+Copiable.generated.swift

```diff
@@ -847,7 +847,7 @@ extension Networking.GoogleAdsCampaign {
         rawStatus: CopiableProp<String> = .copy,
         rawType: CopiableProp<String> = .copy,
         amount: CopiableProp<Double> = .copy,
-        country: CopiableProp<String> = .copy,
+        country: NullableCopiableProp<String> = .copy,
```

Comment:
I think the file wasn't re-generated after some changes in a previous PR, that's why `rage generate` updated it.